libs/labelbox/src/labelbox/schema/internal/datarow_upload_constants.py

```diff
@@ -0,0 +1,4 @@
+MAX_DATAROW_PER_API_OPERATION = 150_000
+FILE_UPLOAD_THREAD_COUNT = 20
```

Comment:
Looks like we also define this in data_row_uploader, do we use this constant anywhere?